README.md

```diff
@@ -4,23 +4,23 @@ This participatory democracy system is built on the City of Helsinki's fork of t
 
 Please check out their README's and documentation for instructions:
 
-* [Decidim](https://github.com/decidim/decidim)
-* [City of Helsinki's modified fork of Decidim](https://github.com/City-of-He<F2>lsinki/decidim-helsinki)
+- [Decidim](https://github.com/decidim/decidim)
```

Comment:
I formatted this file with Prettier (in VS Code) and it replaced `*` with `-`. I hope this change is OK :)